lib/shoulda/matchers/active_record/define_enum_for_matcher.rb

```diff
@@ -111,6 +113,16 @@ def enum_values_match?
           expected_enum_values.empty? || actual_enum_values == expected_enum_values
         end
 
+        def matched_column
+          @_matched_column ||= begin
+            @model.class.columns.detect { |each| each.name == attribute_name.to_s }
+          end
+        end
+
+        def column_type_is_integer?
+          matched_column.sql_type == "integer"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.